templates/server.template.yaml

```diff
@@ -241,11 +249,11 @@ Resources:
                   - 'logs:CreateLogGroup'
                   - 'logs:CreateLogStream'
                   - 'logs:PutLogEvents'
-                Resource: '*'
+                Resource: !Sub arn:${AWS::Partition}:logs:*:*:*
```

Comment:
Can we put specific log group name here, instead of *?

Reply:
Actually, no. We don't create the Log group resource. We rely on Lambda to create it and randomize the name at every run. We have this approach in multiple places throughout this solution. Don't think this is best practice but since this is not a major security flaw, we'd appreciate if you guys would merge this and have the quickstart running again properly.

Reply:
Sure. Can we add this item for the next update to the QS?

Reply:
Will do

Reply:
Is this not addressed yet? I see the last comment was in Aug and was mentioned that it will be addressed in the next update.

Reply:
The next update is the next PR, right? Am I to understand that this is required now??